nav2_bt_navigator/behavior_trees/navigate_w_replanning_and_recovery.xml

```diff
@@ -18,12 +18,15 @@
           <ClearEntireCostmap name="ClearLocalCostmap-Context" service_name="local_costmap/clear_entirely_local_costmap"/>
         </RecoveryNode>
       </PipelineSequence>
-      <SequenceStar name="RecoveryActions">
-        <ClearEntireCostmap name="ClearLocalCostmap-Subtree" service_name="local_costmap/clear_entirely_local_costmap"/>
-        <ClearEntireCostmap name="ClearGlobalCostmap-Subtree" service_name="global_costmap/clear_entirely_global_costmap"/>
-        <Spin spin_dist="1.57"/>
-        <Wait wait_duration="5"/>
-      </SequenceStar>
+      <ReactiveFallback name="RecoveryFallback">
```

Comment:
Can you generate images of these for the PR?

Reply:
I added them in the PR description, should I include them somewhere else? I'm thinking on spending some time going through our documentation for the bt, maybe I could also include them somewhere in the docs?

Reply:
Yes, that could be good. Anywhere that you see a BT diagram that you have now changed in this PR you should update. But it looks like you did that already?

Reply:
I think that's done, yes.